src/PlusImageProcessing/Testing/vtkPlusTransverseProcessEnhancerTest.cxx

```diff
@@ -256,19 +370,19 @@ int main(int argc, char **argv)
 	std::cout << "Done reading Config file." << std::endl;
 
 
-	//test ProcessFrame()
-	LOG_DEBUG("Running ProcessFrame().");
+	//test ProcessFrame method
+	LOG_DEBUG("attempting to Process Frames.");

```

Comment:
Period at the end is optional (we don't use it consistently), but always start log messages with capital letter.